### ISSUE TITLE
Making scan area squared

### ISSUE THREAD

**Issue by ripern** [gruhn/vue-qrcode-reader#31]
Hi, great lib, works fine when not setting :video-constraints, however when
I've tried by first setting aspectRatio: 1 and later
`<qrcode-reader @decode="onDecode" @init="onInit" 
          :video-constraints="{ width: { min: 360, ideal: 360, max: 360 },
          height: { min: 360, ideal: 360, max: 360 } }" />`

I'm just getting a blinking scan area (where I can see it's squared) and 
![image](https://user-images.githubusercontent.com/515357/39298374-497b31a4-4946-11e8-877b-eefdadb3ebea.png)

This was in Chrome 66 in Mac OS.

Any thoughts?

**Comment by gruhn** [comment]
Hmmm... as far I've understood all those `min`, `max`, `aspectRatio` properties are not really meant to force your camera into a specific resolution. This is rather a way to filter the list of installed cameras on your device (in most cases you only have one or two). In the end, the browser selects the camera that matches your constraints best. I don't really know though and since it kind of works for you anyway I'm a little bit confused.

Can you describe your use case a little bit.

Also, I can't reproduce your error. Is it possible that you could share your implementation?

**Comment by globophobe** [comment]
On desktop PC, the camera is in landscape mode. But, on a tablet the camera defaults to portrait mode. 

Unfortunately, with some of the constraints I tried, the camera always worked on desktop, but sometimes not on Android tablets, and not on iPads.

TLDR, The scanning area looks ridiculous in portrait mode. It would be nice if there were some presets that just worked.

**Comment by gruhn** [comment]
I don't get it. Landscape mode on desktop and portrait mode on mobile sounds like reasonable defaults to me. Or do you mean it looks ridiculous when you have your phone in landscape mode?

**Comment by globophobe** [comment]
The constraints really have nothing to do with this repo. 

TBH, I was glad that I could configure it as I like when I chose to make use of it. But, it would be nice to have a nice default that worked across browsers and platforms. 

Landscape mode is OK on all browsers and platforms. Portrait mode not so much.

**Comment by gruhn** [comment]
I still can't really picture those bad cases you describe. You may have some screenshots for me?
